core/mondoo-ms365-security.mql.yaml

```diff
@@ -0,0 +1,611 @@
+policies:
+- uid: mondoo-ms365-security
+  name: Microsoft 365 Security by Mondoo
+  version: 1.0.0
+  scoring_system: 2
+  authors:
+    - name: Mondoo, Inc
+      email: hello@mondoo.com
+  tags:
+    mondoo.com/platform: ms365,cloud
+    mondoo.com/category: security
+  docs:
+    desc: |-
+      ## Overview
+
+      Microsoft 365 Security by Mondoo provides guidance for establishing minimum recommended security and operational best practices for Microsoft 365.
+
+      ## Remote scan
+
+      Remote scans use native transports in `cnspec` to provide on demand scan results without the need to install any agents, or integration. 
+      
+      For a complete list of native transports run: 
+      
+      ```bash
+      cnspec scan --help
+      ```
+      ### Prerequisites
+
+      Remote scans of Microsoft 365 requires API credentials with access to the subscription. 
+
+      ### Scan a Microsoft 365 subscription
+
+      ```bash
+      cnspec scan ms365 --certificate-path <*.pem> --tenant-id <tenant_id> --client-id <client_id> --policy-bundle <*.mql.yaml>
+      ```
+
+      ## Join the community!
+
+      Our goal is to build policies that are simple to deploy, accurate, and actionable. 
+      
+      If you have any suggestions on how to improve this policy, or if you need support, [join the community](https://github.com/orgs/mondoohq/discussions) in GitHub Discussions. 
+  specs:
+  - asset_filter:
+      query: |
+        platform.name == "microsoft365"
+        platform.kind == "api"
+    scoring_queries:
+      mondoo-m365-security-ensure-multifactor-authentication-is-enabled-for-all-users-in-all-roles: null
+      mondoo-m365-security-ensure-that-office-365-passwords-are-not-set-to-expire: null
+      mondoo-m365-security-ensure-multifactor-authentication-is-enabled-for-all-users-in-administrative-roles: null
+      mondoo-m365-security-ensure-that-between-two-and-four-global-admins-are-designated: null
+      mondoo-m365-security-enable-conditional-access-policies-to-block-legacy-authentication: null
+      mondoo-m365-security-ensure-security-defaults-is-disabled-on-azure-active-directory: null
+      mondoo-m365-security-ensure-that-spf-records-are-published-for-all-exchange-domains: null
+      mondoo-m365-security-ensure-that-mobile-devices-require-a-minimum-password-length-to-prevent-brute-force-attacks: null
+      mondoo-m365-security-ensure-that-mobile-device-encryption-is-enabled-to-prevent-unauthorized-access-to-mobile-data: null
+      mondoo-m365-security-ensure-third-party-integrated-applications-are-not-allowed: null
+      mondoo-m365-security-enable-azure-ad-identity-protection-sign-in-risk-policies: null
+      mondoo-m365-security-enable-azure-ad-identity-protection-user-risk-policies: null
+queries: 
+  - uid: mondoo-m365-security-ensure-multifactor-authentication-is-enabled-for-all-users-in-all-roles
+    title: Ensure that multifactor authentication has been enabled for all users
+    severity: 80
```

Comment:
```suggestion
    severity: 60
```

Reply:
Thanks Patrick! Personally believe it should be 100 or 80, but will change to 60 as you requested!

Reply:
you are right

Reply:
Then will change it to 80! Thanks

Reply:
we have decided together for 100!